update/verify_upgrade_db.sh

```diff
@@ -6,14 +6,18 @@ set -e
 # PARAMS
 TESTDB=qwat_test
```

Comment:
can you add a few lines on top to say what the file does (to distinct it from upgrade_db.sh)?